### PR TITLE
Compile Javascript

### PR DESCRIPTION
**Pull request by rardoz**
Organized files.

I had an issue where I was not able to use this package because of the javascript, so I created a compiled version through babel-cli.

I also added better support for react-native-web.

### CODE REVIEW

**Comment by rardoz** [comment]
Anyway we could get this PR through soon?

File: dist/index.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    }
+  }, {
+    key: 'render',
+    value: function render() {
+      var viewProps = _objectWithoutProperties(this.props, []);
+
+      delete viewProps.onPress;
+      delete viewProps.linkDefault;
+      delete viewProps.onLongPress;
+      delete viewProps.linkStyle;
```
Comment (obipawan) [inline dist/index.js]: did these props cause any problems?

**Comment by obipawan** [comment]
Hello @rardoz 

Thanks for doing this! apologies for not getting back earlier - I was on my vacation :)
LGTM - had just one comment.

File: dist/index.js (hunk 1 above)
Comment (rardoz) [reply dist/index.js]: Yeah they did. Mostly warnings with react-native-web. They were being passed to a view and views don't contain those props. So you would get prop warnings.

**Comment by rardoz** [comment]
I added some hooks so the js gets built automatically when you publish or push.

File: dist/index.js (hunk 1 above)
Comment (obipawan) [reply dist/index.js]: ah ok. thanks @rardoz

**Comment by obipawan** [comment]
@rardoz thanks for the build setup and the npm hooks :) much appreciated

**Comment by obipawan** [comment]
available on [v0.0.12](https://github.com/obipawan/react-native-hyperlink/releases/tag/v0.0.12)

**Comment by rardoz** [comment]
Awesome! Thanks @obipawan

**Pull request merged**